packages/AlgernonTests.package/ALGTest.class/instance/assertResultContians..st

```diff
@@ -0,0 +1,3 @@
+helpers
+assertResultContians: aString
```

Comment:
Got a typo here

Reply:
We also already got the `resultsIncludeName:` and `resultsClassIncludeName:` methods. You may want to use one of them. (Or "merge" them with your method)